docs/guides/dark-mode.md

```diff
@@ -0,0 +1,158 @@
+---
+description: How to use Atomizer to enable dark mode on your site.
+layout: docs
+section: docs
+title: Dark Mode
+---
+
+Dark mode has become a standard feature across many websites. OS level support and the [`prefers-color-scheme`] CSS media feature have made it even easier to enable dark mode.
+
+Atomizer does not provide out-of-the-box support for dark mode because it is not required. You can combine the `prefers-color-scheme` CSS preference and the existing [`context`] class support to add dark mode to your website.
+
+Continue reading to learn how to use each concept.
+
+## Browser Basics
+
+All modern browsers [support](https://caniuse.com/prefers-color-scheme) the [`prefers-color-scheme`] CSS media feature. The feature works by defining or overwriting an existing CSS property via the `@media (prefers-color-scheme: dark) {}` directive.
+
+In the example below, we set an initial style for a `div` with the `.intro` class. We overwrite the style when the OS or user-agent preference is set to `dark`.
```

Comment:
maybe we should have this last since it requires `JS` we could show the `prefers-color-scheme` logic first since that is pure css....